src/naive/date.rs

```diff
@@ -1714,6 +1714,22 @@ impl str::FromStr for NaiveDate {
     }
 }
 
+/// The default for a NaiveDate is 1st of January 1970.
+///
+/// # Example
+///
+/// ~~~~
```

Comment:
we recently (in between your submission of this PR and now) swapped all the doctests to use backticks to be more inline with other rust code. If you feel like changing these that'd be appreciated, but your call.

Reply:
I've changed the docs from the `Default` impl to use backticks!